crates/pathfinder/src/rpc/v02/method/add_declare_transaction.rs

```diff
@@ -147,7 +152,8 @@ mod tests {
                         "nonce": "0x0",
                         "contract_class": {},
                         "sender_address": "0x1"
-                    }}
+                    }},
+                    "token": "token"
```

Comment:
Is `token` supported for named params only? If no, should `positional_args` test be also updated?

Reply:
I didn't update that test so that it tests that the parameter is optional. I wasn't very explicit, though...